lib/id3c/labelmaker.py

```diff
@@ -184,6 +184,13 @@ class CollectionsCliaComplianceLayout(LabelLayout):
     reference = "scanpublichealth.org"
 
 
+class SamplesHaarviLayout(LabelLayout):
+    sku = "LCRY-2380"
+    barcode_type = "SAMPLE"
+    copies_per_barcode = 1
+    reference = "HAARVI"
```

Comment:
Looking at the HAARVI barcodes generated last time, the reference was still `seattlelfu.org`. Unclear if this should be the same?

Reply:
Hmm right because it's a sub-study of seattleflu.org... I should ask Caitlin.

Reply:
Caitlin said it's not necessary. HAARVI samples is fine.